src/LogoutForm.test.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import { render, fireEvent } from '@testing-library/react';
+
+import LogoutForm from './LogoutForm';
+
+describe('LogoutForm', () => {
+  const handleClick = jest.fn();
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  const renderLogoutForm = () => render((
+    <LogoutForm
+      onClick={handleClick}
+    />
+  ));
+
+  it('renders “Log out” button', () => {
+    const { container, getByText } = renderLogoutForm();
+
+    expect(container).toHaveTextContent('Log out');
+
+    fireEvent.click(getByText('Log out'));
+
+    expect(handleClick).toBeCalled();
+  });
+
+  context('when button is not clicked', () => {
+    it("doesn't click event handler", () => {
+      renderLogoutForm();
+
+      expect(handleClick).not.toBeCalled();
+    });
+  });
+
+  context('when button is clicked', () => {
+    it('calls click event handler', () => {
+      const { getByText } = renderLogoutForm();
+
+      fireEvent.click(getByText('Log out'));
+
+      expect(handleClick).toBeCalled();
+    });
+  });
```

Comment:
버튼이 안눌렸을때 호출이 안됐다는걸 따로 context 로 분리해서 확인할 필요는 없을 것 같아요.

```suggestion
    it('calls click event handler when click logout button', () => {
      const { getByText } = renderLogoutForm();

      expect(handleClick).not.toBeCalled();  // 이 부분만으로 버튼이 안눌렸을땐 호출되지 않음을 확인 가능할 것 같아요
      
      fireEvent.click(getByText('Log out'));

      expect(handleClick).toBeCalled();
    });
```

Reply:
일요일까지 리뷰 감사합니다!
오옹 context를 굳이 쓰지 않고 좀 더 간단하게 하는 게 다른 동료들이 읽었을 대 좋은 방식 같네요! 감사합니다 ㅎㅎ